src/test/java/org/broadinstitute/hellbender/tools/walkers/conversion/ConversionTestUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.walkers.conversion;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+public class ConversionTestUtils {
+
+    public static String getTestDataDir(){
+        return GATKBaseTest.toolsTestDir + "/walkers/gtfToBed/";
+    }
+
+    public static String getDecoySampleGtf(){
+        return getTestDataDir()+"/decoySample.gtf";
+    }
+
+    public static String getMapk1Gtf(){
+        return getTestDataDir()+"/mapk1.gtf";
```

Comment:
I missed this the first time, but this file is large (3MB). We try to avoid large files over 1MB or so that have to be cloned from ghithub. Its good practice to use git LFS for those files which we have hooked up to a specific file directory. Can you move this into src/test/resources/large directory `largeFileTestDir`?